### ISSUE TITLE
knife node show -a behavior change in 12.3

### ISSUE THREAD

**Issue by christinedraper** [chef/chef#3370]
```knife node show mynode -a override```  

used to show the override priority attribute values set on mynode. With 12.3, this behavior changed and now shows nothing. I realize the original behavior may not have been intentional, but it was quite useful for debugging.

**Comment by btm** [comment]
@christinedraper what version were you using previously?

Testing on 12.0.3:
```
$ knife node show cedar -a override
cedar:
  override:
    bind:
      zonesource: databag
      zonetype:   master
```

12.2.1:
```
$ knife node show cedar -a override
cedar:
  override:
```

12.3.0:
```
$ knife node show cedar -a override
cedar:
  override:
```

**Comment by christinedraper** [comment]
I was using 12.0.3

**Comment by lamont-granquist** [comment]
this works in master, looks like it was fixed by 58410b2f
